docker/README.md

```diff
@@ -31,6 +31,15 @@ docker-compose --profile=services up -d
 docker-compose --profile=reg-loader up
 ```
 
+For test, start the registry with some test data loaded:
+
+    docker-compose --profile=integration-test up
+
+
+For API dev, start the registry with some test data, without the API:
+
+    docker-compose --profile=pre-api-dev up
```

Comment:
I think the Docker people are phasing out `docker-compose` in favor of the new integrated `compose` subcommand for `docker`, i.e., `docker compose`. We may wish to reflect that here.

Reply:
Thanks Sean, I will try that

Reply:
docker compose works the same, I will update the README.